spec/system/ember_deprecation_spec.rb

```diff
@@ -11,13 +11,6 @@
       console.warn = (msg) => window.intercepted_warnings.push([msg, (new Error()).stack])
     JS
 
-    # Apply deprecate shims. These are applied automatically in production
```

Comment:
This test is no longer testing what happens in prod, as we're using the patched Ember.

Is possible/worth it running it using a production build? Otherwise we're really testing if the warnings work in the debug build.

Reply:
Yeah I agree we should just drop the spec.